src/Pact/Types/Term.hs

```diff
@@ -937,13 +940,22 @@ data Term n =
     , _tMeta :: !Meta
     , _tInfo :: !Info
     } |
+    TModRef {
+      _tModRef :: !ModuleName
```

Comment:
🎉 

Let's use it! Also I think we probably want to record the interface names, just to have parity with the `TyModule` structure.